projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -523,6 +529,32 @@ export const getStepContent = ( state, stepSlug ) => {
 				illustration: 'assistant-backup-welcome',
 				skipText: __( 'Next', 'jetpack' ),
 			};
+		case 'welcome__social_advanced':
+			return {
+				question: __( 'Welcome to Jetpack Social!', 'jetpack' ),
+				description: __(
+					"With your new advanced plan you unlocked unlimited sharing, access to upload photos and videos with your posts, and usage of Social Image Generator.<br/><br/>Let's start with connecting your social media accounts, if you haven't already.",
+					'jetpack'
+				),
+				ctaText: __( 'Manage Social Media Connections', 'jetpack' ),
+				ctaLink: getRedirectUrl( 'calypso-marketing-connections', {
+					site: getSiteRawUrl( state ),
+				} ),
+				illustration: 'assistant-jetpack-social',
+				skipText: __( 'Next', 'jetpack' ),
+			};
+		case 'welcome__social_image_generator':
+			return {
+				question: __( 'Social Image Generator', 'jetpack' ),
+				description: __(
+					'Create beautiful social media previews for your posts with Social Image Generator. You can customize the text, image, and template to match your brand.<br/><br/>You can turn on Social Image Generator for individual posts from the post editor, or turn it on by default for all future posts from the Jetpack Social Settings.',
```

Comment:
Nitpick: should it be `from Jetpack Social Setttings` instead of `from the Jetpack Social Settings`?